### ISSUE TITLE
G3000 Map Freeze

### ISSUE THREAD

**Issue by diaznet** [Working-Title-MSFS-Mods/fspackages#1279]
Hello

Odd issue with g3000-v0.7.4a in TBM930 (have not tried elsewhere) and SU5.
The map on MFD (center screen) remains frozen, unless I zoom all the way in (max zoom in). It then refreshes immediately but also freezes again immediately.
VFR map doesn't have this issue at all.

![image](https://user-images.githubusercontent.com/35642729/127537344-24d0dc1a-0c3d-4f4a-9a00-a0632784eb51.png)

**Comment by SPAREGARLIC556** [comment]
I'm seeing the same problem you can get the plane icon and map to move if you click on the Traffic button then click back on the Map button. It is post SU5 all good before. The plane TBM930.

**Comment by SPAREGARLIC556** [comment]
I'm seeing the same problem you can get the plane icon and map to move if you click on the Traffic button then click back on the Map button. It is post SU5 all good before. The plane TBM930.

**Comment by coleoptere91** [comment]
I confirm 
NavMap freeze. Zooming actualize aircraft position but freezes again. So NavMap is not usable !

**Comment by coleoptere91** [comment]
I have to say that mys MSFS General Option Accesibility is set to "Legacy" and not "Lock"

**Comment by Porkki** [comment]
I have exact same issue, started right after when I installed TBM930 Improvement mod. I will test shortly if removing that resolves the issue and report here.

**Comment by Porkki** [comment]
> I have exact same issue, started right after when I installed TBM930 Improvement mod. I will test shortly if removing that resolves the issue and report here.

Quick update, i am currently on flight and the freezing issue starts when navmap is zoomed more than 1nm shown in screen

**Comment by coleoptere91** [comment]
Ok
That’s exactly what I found out too.
To be debugged

**Comment by Porkki** [comment]
Yep, removing the TBM930 Improvement mod fixed this issue for me. They seem to be conflicting somehow.

**Comment by coleoptere91** [comment]
You confirm TBM930 improvment or GitHub Working Title G3000 add on ?

**Comment by sand3r** [comment]
Can confirm: just had the same (again) with the Longitude + G3000. Most flights are fine, but now and then it seems to freeze. I do have the TBM 930 improvement mod as well but already installed for a longer time and does not always happen.

**Comment by Richdem** [comment]
Just wondering if anyone has found a fix for the map freezing?

**Comment by coleoptere91** [comment]
If you have it it your Community folder remove
"TBM930 improvment"
I did it and no more any NavMap freeze

**Comment by sand3r** [comment]
> If you have it it your Community folder remove
> "TBM930 improvment"
> I did it and no more any NavMap freeze

It may be related, but out of ~25 flights I had just 2 map freezes. All flights with the Longitude. TBM930 improvement mod was installed. So again, it may be related, but is most definitely not the root cause. :)

**Comment by coleoptere91** [comment]
OK, but 25 **Longitude**  flights... no TBM930 flights.
May be others parameters to take in account .

**Comment by coleoptere91** [comment]
A lot of TBM flights without any MFD NavMap freeze. But today freeze is back on 2 flights. No change in my MSFS configuration
Random BUG ?
Hope it will be fixed

**Comment by Lysdestic** [comment]
I'm seeing this same issue. No TBM Improvement, default livery, and I have removed all other working title mods and issue still persists.

**Comment by coleoptere91** [comment]
can we expect to have this problem resolved next weeks...months...

**Comment by ByronNF** [comment]
I still have this problem with the G3000 map freezing (g3000-v0.7.7) in the TBM. I'm not using any other mods at the moment. Did anyone find out what's causing this bug?

**Comment by coleoptere91** [comment]
When the MFD NavMap  freeze, I display Trafic Map or Weather Map and then when going back to Nav Map that one is updated but immediatly again frozen
Then whta you can see is taht Up TCA banner is not updated and display Weather Map or Traffic Map according to waht was your last action.  Should indicate Nav Map !!!
Again
can we expect to have this problem resolved next weeks...months..

**Comment by mattnischan** [comment]
As the G3000 mod is in a maintenance/compatibility updates only state (as all G3000 effort is focused on the upcoming Marketplace refresh version) we cannot commit to any kind of fix timeline, and I would say that a fix from us specifically is unlikely, as we cannot presently reproduce the issue.

However, as always, any member of the community is welcome to push a fix for this issue should they find one.

**Comment by sand3r** [comment]
Just reinstalled the G3000 and made a flight with the Longitude. Right away the map froze again. I can only get the map to move if I zoom in quite a bit (decrease range) to 5 nm.

Reading now that it won't be fixed, unfortunately. Pity it can't be reproduced because quite a few have reported this issue now.

**Comment by Lysdestic** [comment]
> Just reinstalled the G3000 v0.7.7 and made a flight with the Longitude. Right away, this time even before take-off, the map froze again. I can only get the map to move if I zoom in quite a bit (decrease range) to 5nm (later in the flight even 2.5nm).
> 
> Reading now that it won't be fixed, unfortunately. Pity it can't be reproduced because quite a few have reported this issue now.

It's not that it can't be reproduced, it's that the G3000 is now in maintenance only and Working Title has stated that they will not be doing any significant G3000 work until they do the refresh a la NXi for the G1000 as part of their partnership with Asobo.

Some users on discord have had luck by setting useCustom = false within the WTg3000.cfg file. YMMV.

**Comment by ByronNF** [comment]
> > Just reinstalled the G3000 v0.7.7 and made a flight with the Longitude. Right away, this time even before take-off, the map froze again. I can only get the map to move if I zoom in quite a bit (decrease range) to 5nm (later in the flight even 2.5nm).
> > Reading now that it won't be fixed, unfortunately. Pity it can't be reproduced because quite a few have reported this issue now.
> 
> It's not so much that it can't be reproduced, if they were actively looking at it, I'm sure it could as we've all encountered it at one point or another. It's that the G3000 is now in maintenance only and Working Title has stated that they will not be doing any significant G3000 work until they do the refresh a la NXi for the G1000 as part of their partnership with Asobo.
> 
> Some users on discord have had luck by setting useCustom = false within the WTg3000.cfg file. YMMV.

The "useCustom = false" worked for me.
Thanks.

**Comment by coleoptere91** [comment]
I confirm , It worked for me also
BUT
Now, the VFR Map is reduced to a little square in the left window corner  !!!
You may resize le VFR Map Window, but the map stay as a little square !
